crates/agglayer-storage/src/columns/latest_certificate_per_network/mod.rs

```diff
@@ -1,7 +1,7 @@
+use agglayer_types::NetworkId;
```

Comment:
Maybe also using the types defined in `agglayer-types` for the  `ProvenCertificate` defined in this file below?

```
pub struct ProvenCertificate(pub [u8; 32], pub u64, pub u64);
```